### ISSUE TITLE
Quest: Badlands Reagent Run II

### ISSUE THREAD

**Issue by underlord-oberon** [plantain-gnocchis/wow-voiceover-tbc-wotlk#58]
Add-on version: AI_VoiceOver_v1.4.5
Data version: AI_VoiceOverData_VanillaExtra_v1.4

Badlands / Kargath / NPC: Jarkal Mossmeld / Quest: Badlands Reagent Run II
Link: https://www.wowhead.com/wotlk/quest=2203/badlands-reagent-run-ii

Description: the voice-over suddenly silences during playback.

Progress: no playback.

Completion: playback ok.

**Comment by carevass** [comment]
This should be fixed in AI_VoiceOverData_VanillaExtra_v1.5.
